CONTRIBUTING.md

```diff
@@ -123,6 +119,63 @@ use a shared working branch strategy.
 5. Resolve any merge conflicts.
 6. When your work is complete, merge the shared working branch into `develop` via a PR.
 
+### Working with docs-website-private
+
+The [docs-website-private repository](https://github.com/newrelic/docs-website-private) is a private copy of the docs-website repository. It is meant for private colloboration, on work that is not ready for public consumption.
+
+If you need to interact with the public & private repositories, there are three setups that you can use:
+1. cloning docs-website, adding a remote to the private repository.
+2. cloning docs-website-private, adding a remote to the public repository.
+3. doing both of the above.
+
+Our recommendation is setup #1, but you can use #2 or #3 to do the same things.
+
+#### General Workflow
+
+The general workflow is outlined [here](https://docs.google.com/document/d/1Lk16HGkWHlf52icq8AmaAiUp-lzFHEULCP7RAVjJ4aY/edit), and repeated below:
+
+1. Clone `docs-website`.
+   ```sh
+   git clone https://github.com/newrelic/docs-website.git
+   ```
+2. Add `docs-website-private` as a Remote.
+   ```sh
+   cd docs-website
+   git remote add private https://github.com/newrelic/docs-website-private.git # add a new remote called 'private'
+   ```
+3. Work on Private Content. Depending on the nature of the work & interaction of collaborators, using a [shared working branch](#shared-working-branches) may be valuable.
+   ```sh
+   git checkout develop # starting from develop
+   git pull # pull in most recent changes
+   git checkout -b shared_working_branch # cut your branch from develop
+   git push --set-upstream private shared_working_branch # push up the branch to the private repository for others to access
+   ```
+4. Once private work is ready to become public, push it to the public repository.
+   ```sh
+   git push --set-upstream origin shared_working_branch
+   ```
+5. From here, you can PR the branch in the public repository as normal.
+
+**Note**: since this workflow is across two repositories, if you are focused on your work in the private repository, it may be easy to forget that changes are being made on the public repository and that these changes may conflict with changes you are making.
```

Comment:
I think we can  be more explicit here and just tell users to keep their branches in sync as it's a good practice.